lib/nrk-ellipsis--active.svg

```diff
@@ -1 +1,3 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" fill="currentColor" d="M23 12c0-6.075-4.925-11-11-11S1 5.925 1 12s4.925 11 11 11 11-4.925 11-11zM6 10a2 2 0 110 4 2 2 0 010-4zm6 0a2 2 0 110 4 2 2 0 010-4zm6 0a2 2 0 110 4 2 2 0 010-4z"/></svg>
+<svg  viewBox="0 0 24 24"  xmlns="http://www.w3.org/2000/svg">
```

Comment:
I was asked to have a look at this PR. Looks like there are two spaces ahead of `viewBox`.

Reply:
thanks, was that all?

Reply:
I have little experience with converting svgs and have no previous knowledge of the repo. That combo makes it a bit hard for me to spot errors, I guess (if there are any) :)

Reply:
thanks anyway, lgtm..